src/app/shop/[category]/[productId]/_components/TabContents/ReturnExchangeInfo.module.scss

```
@@ -0,0 +1,49 @@
+.table {
+  @include pretendard-16-400;
+
+  word-break: keep-all;
+
+  tr {
+    border-bottom: 0.1rem solid $gray-10;
+  }
+
+  td {
+    padding: 1.6rem 2rem;
+  }
+
+  span {
+    padding-left: 1rem;
+    @include pretendard-16-400;
+
+    color: $gray-50;
+  }
+}
+
+.title-wrap {
+  @include flex-column(1.2rem);
+
+  padding: 4rem 0;
+  border-top: 0.1rem solid $gray-10;
+  border-bottom: 0.1rem solid $gray-10;
+  background-color: $gray-5;
+  text-align: center;
+
+  @include pretendard-20-700;
```

```suggestion
  @include flex-column(1.2rem);
   @include pretendard-20-700;

  padding: 4rem 0;
  border-top: 0.1rem solid $gray-10;
  border-bottom: 0.1rem solid $gray-10;
  background-color: $gray-5;
  text-align: center;
```

저스트 요렇게 있어도 좋을 것 같다는 저의 의견..!!